projects/playground/.eslintrc.json

```
@@ -0,0 +1,23 @@
+{
+    "extends": "../../.eslintrc.json",
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts"],
+            "parserOptions": {
+                "project": [
+                    "projects/playground/tsconfig.app.json",
+                    "projects/playground/tsconfig.spec.json",
+                    "projects/playground/e2e/tsconfig.json"
+                ],
+                "createDefaultProgram": true
+            },
+            "plugins": ["sonarjs", "eslint-plugin-rxjs"],
+            "rules": {}
+        },
+        {
+            "files": ["*.html"],
+            "rules": {}
+        }
```

Since we're not overriding any rules, we should drop this

```suggestion
        }
```